src/app/static/src/app/components/versions/Versions.vue

```diff
@@ -1,21 +1,41 @@
 <template>
     <div>
-        <div class="col-6">
-            <h2 v-translate="'versionsHeaderCreate'"></h2>
-            <button type="button"
-                    class="btn btn-red mt-2"
-                    @click="createFakeVersion"
-                    v-translate="'createVersion'">
-            </button>
+        <div v-if="!loading" id="versions-content" >
+            <div class="versions-header">
+                <span v-translate="'versionsHeaderCreate'"></span>
+                <span v-if="currentVersion">
+                    <span v-translate="'or'"></span>
+                    <a v-translate="'versionsHeaderReturn'"
+                       href="#" @click="setManageVersions(false)"></a> ({{currentVersion.name}})
+                </span>
+            </div>
+            <div class="my-3 col-6 clearfix">
```

Comment:
There is some weird padding going on here and it's because this `col-6` is not wrapped in a parent `row`. I would suggest adding the `row` class to the div with id `versions-content`, then giving `versions-header` a `col-12` class.